<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cadastroCliente</class>
 <widget class="QDialog" name="cadastroCliente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>832</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	background-color:white;
}

QLineEdit{
	border:1px solid gray;
	border-radius: 6px;
	padding-left: 15px;
	height: 35px;

}</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>481</width>
     <height>741</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_1">
    <item>
     <widget class="QLabel" name="Adicionar_Cliente">
      <property name="font">
       <font>
        <family>Montserrat ExtraBold</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(99, 99, 99);</string>
      </property>
      <property name="text">
       <string>Adicionar Cliente</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Montserrat Medium</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Nome Completo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nome">
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Montserrat Medium</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>RG</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="rg">
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>Montserrat Medium</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>CPF</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="cpf">
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="font">
         <font>
          <family>Montserrat Medium</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Telefone</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="telefone">
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="font">
         <font>
          <family>Montserrat Medium</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>E-mail</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="email">
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="font">
         <font>
          <family>Montserrat Medium</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Endere√ßo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="endereco">
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>780</y>
     <width>211</width>
     <height>43</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnCadastrarCliente">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>41</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #34D481;
	color: white;
	border: none;
	border-radius: 8px;
	font-weight: bold;
	font-size: 15px
}</string>
      </property>
      <property name="text">
       <string>Cadastrar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelar">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>41</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #585858;
	color: white;
	border: none;
	border-radius: 8px;
	font-weight: bold;
	font-size: 15px
}</string>
      </property>
      <property name="text">
       <string>Cancelar</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelar</sender>
   <signal>toggled(bool)</signal>
   <receiver>cadastroCliente</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>473</x>
     <y>794</y>
    </hint>
    <hint type="destinationlabel">
     <x>599</x>
     <y>756</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
